examples/resnet50/resnet50.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright 2022 Advanced Micro Devices, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GUARD_EXAMPLES_RESNET50_RESNET50
+#define GUARD_EXAMPLES_RESNET50_RESNET50
+
+#include <cxxopts/cxxopts.hpp>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace fs = std::filesystem;
+
+struct Args {
+  fs::path path_to_model;
+  fs::path path_to_image;
+  fs::path path_to_labels;
+  uint16_t http_port = 8998;
+  uint16_t grpc_port = 50'051;
```

Comment:
Is that a typo?

Reply:
It's a [digit separator](https://riptutorial.com/cplusplus/example/31798/digit-separator)